requirements.txt

```diff
@@ -32,3 +32,12 @@ snowballstemmer==1.2.0
 traitlets==4.0.0
 websocket-client==0.34.0
 wsgiref==0.1.2
+
```

Comment:
The issue is that the current installation method deprecates use of the requirements.txt in favor of extras_require in setup.py.  Using something like this should accomplish this (not sure how it differs between a core and an extra requirement):
https://mike.zwobble.org/2013/05/adding-git-or-hg-or-svn-dependencies-in-setup-py/